server.js

```diff
@@ -1,35 +1,107 @@
+/* eslint-disable linebreak-style */
 import express from 'express'
-import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
+import listEndpoints from 'express-list-endpoints'
+import dotenv from 'dotenv'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
+import exercisesData from './data/exercises.json'
```

Comment:
I really like the fact that you're taking the time to create a dataset of your own. Great job!